teuthology/orchestra/remote.py

```diff
@@ -206,6 +206,20 @@ def run(self, **kwargs):
         r.remote = self
         return r
 
+    def mkdtemp(self, suffix='', prefix='', dir=None):
```

Comment:
We should avoid using python on the remote nodes for such kind of functions, because any python version can be not present on the remote system.
Why not using system `mktemp -d` from coreutils?

Reply:
What about this two-line code?
```
mkdtemp(self, path='/tmp/tempdir'):
   return self.sh("mktemp -d {}.XXXX".format(path)).strip()
```

Reply:
> What about this two-line code?
> 
> ```
> mkdtemp(self, path='/tmp/tempdir.XXXX'):
>    return self.sh("mktemp -d {}".format(path)).strip()
> ```

Just added some code. :P